tests/backend/core/Core/Functional/KeycloakUserDataMapperTest.php

```diff
@@ -0,0 +1,193 @@
+<?php
+declare(strict_types=1);
+
+namespace Tests\Core\Core\Functional;
+
+use DemosEurope\DemosplanAddon\Contracts\Entities\UserInterface;
+use demosplan\DemosPlanCoreBundle\DataFixtures\ORM\TestData\LoadCustomerData;
+use demosplan\DemosPlanCoreBundle\DataFixtures\ORM\TestData\LoadUserData;
+use demosplan\DemosPlanCoreBundle\Entity\User\AnonymousUser;
+use demosplan\DemosPlanCoreBundle\Entity\User\User;
+use demosplan\DemosPlanCoreBundle\EventDispatcher\EventDispatcherPostInterface;
+use demosplan\DemosPlanCoreBundle\Logic\KeycloakUserDataMapper;
+use demosplan\DemosPlanCoreBundle\Logic\User\CustomerService;
+use demosplan\DemosPlanCoreBundle\Logic\User\OrgaService;
+use demosplan\DemosPlanCoreBundle\Logic\User\RoleHandler;
+use demosplan\DemosPlanCoreBundle\Logic\User\UserService;
+use demosplan\DemosPlanCoreBundle\ValueObject\KeycloakUserData;
+use Liip\FunctionalTestBundle\Test\WebTestCase;
+use Psr\Log\NullLogger;
+use Stevenmaguire\OAuth2\Client\Provider\KeycloakResourceOwner;
+use Tests\Base\FunctionalTestCase;
+use Tests\Base\MockMethodDefinition;
+
+class KeycloakUserDataMapperTest extends FunctionalTestCase
+{
+
+    private ?KeycloakUserDataMapper $keycloakUserDataMapper;
+
+    /**
+     * @var User
+     */
+    protected $citizenUser;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
```

Comment:
setup failed here

Reply:
works for me, we need to check your setup. What fails?

Reply:
Okay if it works, then I have to check my setup.